tests/select/test_drop_over_night_pv.py

```diff
@@ -3,3 +3,5 @@
 
 def test_drop_night_pv(simple_netcdf_datapipe):
     night_drop_pv = DropNightPV(simple_netcdf_datapipe)
+    data = len(next(iter(night_drop_pv)))
```

Comment:
This is a good step! But I would prefer checking the data values of the output, not just the length, but the actual content of the output, if possible.